rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -903,6 +903,39 @@ void getNextFederationCreationBlockHeight_postIris300AndNoValueInStorage_returns
 
     }
 
+    @ParameterizedTest
+    @MethodSource("provideFederationBtcUTXOsTestArguments")
+    void saveNewFederationBtcUTXOs_utxosShouldBeSavedToStorage(FederationStorageIndexKey federationStorageIndexKey, NetworkParameters networkParameters, ActivationConfig.ForBlock activations) {
+
+        // Arrange
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
```

Comment:
Could this line be moved to the setup?

Reply:
There's no `setup` in this test class yet. I could create a new pr to only add that to all the tests to avoid doing so many rebases and changes to the individual prs. What do you think?